packages/ffe-searchable-dropdown-react/less/searchable-dropdown.less

```diff
@@ -54,6 +54,8 @@
     }
 
     &__button {
+        background: @ffe-white;
```

Comment:
You're declaring `background` again as `transparent` on line 58/60 so I'm not sure you need this?

Reply:
Gott observert. Har amandat in endringen i lessfilen eftersom det den er veldigt liten. 

```
@button-width: 40px;
.....
 &--expanded .ffe-dropdown {
        background-image: none;
        padding-right: calc(@button-width + 3px);
    }
```